examples/configs/pose_graph/pose_graph_synthetic.yaml

```diff
@@ -4,8 +4,8 @@ profile: True
 savemat: True
 
 num_poses: 256
-rotation_noise: 0.05
-translation_noise: 0.02
+rotation_noise: 0.02
+translation_noise: 0.05
```

Comment:
Hi @suddhu, looks like you switch the noise ratio for rotation and translation. Could you please use  `rotation_noise: 0.05` and `translation_noise: 0.02`? Thanks!

Reply:
You might also need to flip this line https://github.com/facebookresearch/theseus/blob/310a87643394b3b624b82d21ac0617cc7e86dafd/theseus/utils/examples/pose_graph/dataset.py#L276